forge-game/src/main/java/forge/game/ability/effects/PlayEffect.java

```diff
@@ -509,7 +513,7 @@ protected void addIllusionaryMaskReplace(Card c, SpellAbility sa, Map<AbilityKey
         final Card hostCard = sa.getHostCard();
         final Game game = hostCard.getGame();
         final Player controller = sa.getActivatingPlayer();
-        final String name = hostCard.getName() + "'s Effect";
+        final String name = hostCard + "'s Effect";
```

Comment:
i'm pretty sure that would crash

Reply:
It will use `toString()` which provides the translated name instead of english only